### DISCUSSION TITLE
Official Roborock integration out ??!?

### DISCUSSION THREAD

**Discussion by guanaco0403** [humbertogontijo/homeassistant-roborock#282] [Q&A]
Hello, what is this new roborock official integration introduced in home assistant core update 2023.5 ??? is that your integration and does it have more or less functionalities ?? because i dont want to remove yours just to test it i have too much things configured with it
https://www.home-assistant.io/blog/2023/05/03/release-20235/
https://www.home-assistant.io/integrations/roborock

**Comment by guanaco0403** [comment]
btw i really like your integration i made some clean reactive dashboards super cool 
![image](https://user-images.githubusercontent.com/76651037/236158434-3bff30bc-d39b-413d-b48f-c038cc9d2757.png)

**Reply by Propriide** [reply]
how you extract the map? i do not have any Xiaomi apps, only using roborock atm. your dashboard looks very clean!

**Reply by guanaco0403** [reply]
Thx, the map is à part of this roborock integration, just install this integration via hacs and connect to your roborock account then go to settings-> devices and services -> click on your roborock integration "devices" and import the map to dashboard

**Comment by Lash-L** [comment]
Less functionalities for now. As of right now I'm the core primary dev and humbertogontijo is the custom component primary dev. But we both do work on the custom component and our underlying api. And humbertogotijo will work on core as well in the future I believe

**Reply by guanaco0403** [reply]
Oh ok soo u recomend me to stay on hacs version

**Reply by Lash-L** [reply]
For now if you already have it - yeah i do

**Reply by rleongcs** [reply]
@Lash-L Will the custom component be deprecated once the core is on par? Or custom component will always have more functionalities?

**Reply by Lash-L** [reply]
@rleongcs its highly unlikely this repo will ever be depreciated. 

But also I wouldn't say this repo will always have more functionality. 

The core integration has more users so I would say once they reach feature parity, core would take priority. But hacs is always a good way to test new features and bug fixes

**Reply by rleongcs** [reply]
Thanks...Would you be able to mention that as part of the release notes once we are there so users like me can make the switch 😏

**Reply by andras-tim** [reply]
The same here: [what's next after 2023.5 #296](https://github.com/humbertogontijo/homeassistant-roborock/issues/296)

**Comment by Fuchschr11** [comment]
After updating HA the new integration is available. My old integration with HACS says:

Retrying setup: unsupported operand type(s) for +: 'int' and 'list'

When adding Roborock native integration, it says "Account already configured" after entering login credentials.

What to do?

**Reply by Fuchschr11** [reply]
ok, after updating to latest HA - Version and rebooting, it works apparently again.
